README.md

```diff
@@ -4,4 +4,51 @@
 # MicroJPA
 
 MicroJPA is a framework to provide injection of resource-local ``EntityManager``s and their factories via ``@PersistenceContext`` and ``@PersistenceUnit`` in CDI-environments where no such injection is present.
-That may be the case in testing scenarios or with servers that do not implement the full JEE stack like [Meecrowave](https://openwebbeans.apache.org/meecrowave/).  
\ No newline at end of file
+That may be the case in testing scenarios or with servers that do not implement the full JEE stack like [Meecrowave](https://openwebbeans.apache.org/meecrowave/).  
+
+## Latest Release
+
+Latest release version: `0.9.5`
+
+To use it in a Maven project the following dependency should be added to the pom.xml:
+```xml
+<dependency>
+  <groupId>org.microjpa</groupId>
+  <artifactId>microjpa</artifactId>
+  <version>0.9.5</version>
+</dependency>
+```
+
+or when using Gradle the following line needs to be added to the dependencies block:
+
+```groovy
+implementation 'org.microjpa:microjpa:0.9.5'
+```
+
+The version listed here may not always be up-to-date. It can be checked on [Maven Central](https://search.maven.org/artifact/org.microjpa/microjpa).
```

Comment:
Please remove this section. We should actually aim to always be up-to-date with the versions

Reply:
Done.